metrics-micrometer/README.md

```diff
@@ -0,0 +1,21 @@
+Zipkin Reporter Metrics by Micrometer
+=====
+
+This module provides an implementation of Zipkin Reporter's `ReporterMetrics` for [Micrometer](https://micrometer.io).
+
+Usage
+---
+
+Pass an instance of `MicrometerReporterMetrics` to the `AsyncReporter` builder like below.
+
+```java
+reporter = AsyncReporter.builder(sender).metrics(MicrometerReporterMetrics.create(meterRegistry)).build();
+```
+
+### Extra tags
+
+You may optionally configure Micrometer tags to add to all reporter metrics created using the builder.
```

Comment:
It would probably be good to mention here the one use case we're aware of for extra tags: Zipkin service name as a tag if not already present.

Reply:
Added. Please check.

Reply:
sounds reasonable